test/test_mutant_epitope_predictor.py

```diff
@@ -21,6 +21,9 @@
 from topiary import MutantEpitopePredictor, epitopes_to_dataframe
 from varcode import Variant, VariantCollection
 
+ensembl.download()
```

Comment:
Conversely, it gives us less control over where hefty jobs run. Let's keep
it for now as is but possibly revisit if other Ensembl releases become
problematic for Topiary.

On Wed, Aug 26, 2015 at 1:52 PM, Tavi Nathanson <notifications@github.com>
wrote:

> In test/test_mutant_epitope_predictor.py
> <https://github.com/hammerlab/topiary/pull/25#discussion_r38012710>:
>
> > @@ -21,6 +21,9 @@
> >  from topiary import MutantEpitopePredictor, epitopes_to_dataframe
> >  from varcode import Variant, VariantCollection
> >
> > +ensembl.download()
>
> I had that as the first commit, but felt that logic should be coupled with
> where it's used. When coupled with the test using it, no one needs to
> "remember" to add some random script.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/topiary/pull/25/files#r38012710>.
>